locations/spiders/google_offices.py

```diff
@@ -0,0 +1,25 @@
+import scrapy
+
+from locations.categories import Categories, apply_category
+from locations.dict_parser import DictParser
+
+
+class GoogleOfficesSpider(scrapy.Spider):
+    name = "google_offices"
+    allowed_domains = ["about.google"]
+    item_attributes = {
+        "brand": "Google",
+        "brand_wikidata": "Q95",
+        # Skip NSI matching as we don't want to assign operator tags to Google offices
+        "nsi_id": "N/A",
```

Comment:
Will be fixed soon
https://github.com/osmlab/name-suggestion-index/commit/3fd6abe180277c09bda6cf5ed4671918b596b75e